index.html

```diff
@@ -954,14 +955,19 @@ <h2>
           promise will be resolved when the orientation will change as
           described in the <a>Screen orientation change algorithm</a>.
         </div>
+      </section>
+      <section>
+        <h2>
+          <dfn>Locking the orientation algorithm</dfn>
```

Comment:
Like with the algorithm above, this algorithm should be defined in the paragraph below. Then the user agent runs the algorithm.

Reply:
At the moment it is defined in the line below with 

> When the user agent has to <dfn>lock the orientation</dfn> of a document to orientations, it MUST run the following steps: 

Do you want something additional there?